python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -529,6 +530,23 @@ def value_to_storage_dict(
         # job_name as pipeline_name
         return replace_storage_keys(storage, {"job_name": "pipeline_name"})
 
+    @classmethod
+    def value_from_storage_dict(
```

Comment:
where do we actually store `RunsFilter`?

Reply:
I guess we could just have it deserialized in cloud from a client running a previous version of dagster...

might be worth adding a comment that this is specifically for that case, and note that we could drop this if we can guarantee that all clients are running version `0.14.7`